### DISCUSSION TITLE
How do you use the projectors?

### DISCUSSION THREAD

**Discussion by spike11302000** [fnuecke/oc2#119] [Q&A] [answered]
I see that projectors where added in the recent commit but theres not documentation on how to use it also is the projector limited to only lua or will it act like a frame buffer in linux?

**Comment by fnuecke** [comment, marked as answer]
They are indeed framebuffer devices. I'll write the manual entry when I find the time.

Basically: connect projector via bus interface, power it somehow. And you may want to connect a keyboard, to use the tty on the framebuffer/projector directly. Alternatively you can also throw r5g6b5 to `/dev/fb0` directly.

**Reply by spike11302000** [reply]
Nice. Thanks for the help
